docs/clusterpool.md

```diff
@@ -0,0 +1,28 @@
+[comment]: # ( Copyright Contributors to the Open Cluster Management project )
+
+# ClusterPools
+
+The CLI has commands to manage clusterpools. It allows to manage multiple clusters that are holding the clusterpool management system. 
+
+These clusters are called here `clusterpoolhost`
+
+## Manage ClusterPoolHosts
+
+A clusterpoolhost can be created using the command `cm create clusterpoolhost|cph <clusterpoolhost_name> <options>`. 
+
+The list of clusterpoolhosts can be retreived by calling the command `cm get clusterpoolhosts|cphs <options>`.
+
+The list of clusterpoolhosts is maintained in the `./kube/known-cphs`.
```

Comment:
This should be `~/.kube/known-cphs` right?

Reply:
Changed will be part of the next commit